tests/unit/core/test_installAddon.js

```diff
@@ -885,6 +886,26 @@ describe(__filename, () => {
         handler({ state: 'WAT' }, { type: 'onNothingPerformed' });
         sinon.assert.notCalled(dispatch);
       });
+
+      it('sets status to error on onCorruptFile', () => {
+        const dispatch = sinon.spy();
+        const guid = '{my-addon}';
+        const name = 'my-addon';
+        const type = ADDON_TYPE_EXTENSION;
+        const handler = createProgressHandler({
+          dispatch,
+          guid,
+          name,
+          type,
+        });
+
+        handler({ state: 'STATE_SOMETHING' }, { type: 'ERROR_CORRUPT_FILE' });
```

Comment:
I believe you can extract `type` into its own variable as it is used below too.

Reply:
I was thinking about extracting `type: 'ERROR_CORRUPT_FILE'` instead of the addon's type.